amd_openvx_extensions/amd_rpp/source/kernel_rpp.cpp

```diff
@@ -2757,6 +2802,46 @@ void fillAudioDescriptionPtrFromDims(RpptDescPtr &descPtr, size_t *tensorDims) {
     descPtr->numDims = 4;
 }
 
+void fillGenericDescriptionPtrfromDims(RpptGenericDescPtr &dscPtr3D, vxTensorLayout layout, size_t *tensorDims) {
+    switch(layout) {
+        case vxTensorLayout::VX_NDHWC: {
+            dscPtr3D->numDims = 5;
```

Comment:
This function should handle for other layouts like NHWC, NCHW and other inputs where there is no layout associated
Please check and change

Reply:
Can we add the other layouts as we keep add those test case? I will not be able to test it now.
Please comment on this @sampath1117